cpp/src/structs/utilities.cpp

```diff
@@ -322,7 +322,7 @@ void superimpose_parent_nulls(bitmask_type const* parent_null_mask,
       reinterpret_cast<bitmask_type const*>(parent_null_mask),
       reinterpret_cast<bitmask_type const*>(current_child_mask)};
     std::vector<size_type> begin_bits{0, 0};
-    cudf::detail::inplace_bitmask_and(
+    [[maybe_unused]] auto const null_count = cudf::detail::inplace_bitmask_and(
```

Comment:
@nvdbaranec I tried to use this null count below, instead of `UNKNOWN_NULL_COUNT`, but it causes failures in the Python test `test_parquet_reader_struct_sol_table`. Do you have any ideas for why this might be occurring?

Reply:
@nvdbaranec Sorry for the false alarm, I realized my mistake right after posting the question. "Rubber duck debugging" was what I needed. This is resolved in d1b90f9. :duck: